### PR TITLE
feat(handlers): add createIntegration()

### PR DESCRIPTION
**Pull request by itohatweb**
**Please describe the changes this PR makes and why it should be merged:**

**Status**

- [ ] Code changes have been tested against the Discord API, or there are no code changes

**Semantic versioning classification:**

- [x] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: src/api/handlers/guild.ts [hunk 1]
```diff
@@ -602,6 +602,21 @@ export async function getIntegrations(guildID: string) {
   return RequestManager.get(endpoints.GUILD_INTEGRATIONS(guildID));
 }
 
+/** Attach an integration form the current user to the guild */
```
Comment (ayntee) [inline src/api/handlers/guild.ts:605]: "Requires X permission"

**Comment by Skillz4Killz** [comment]
This endpoint cant be used by bots and DD doesnt support self bot usage

**Comment by Skillz4Killz** [comment]
@itohatweb If this endpoint is already in constants lets already remove that plz

**Comment by itohatweb** [comment]
Gonna remove it

**Pull request closed without merging**